src/models.ts

```diff
@@ -124,6 +124,7 @@ export interface IFilterMeasureTarget extends IBaseFilterTarget {
 }
 
 export declare type IFilterTarget = (IFilterColumnTarget | IFilterHierarchyTarget | IFilterMeasureTarget);
+export declare type IBaseFilter = (IBasicFilter | IAdvancedFilter);
```

Comment:
The name `Base` was proposed when this was the base interface which was extended by IBasicFilter and IAdvancedFilter. As it is now, it's not a base of anything just a union of two filters.  I like the idea of this being non-breaking change by only adding instead of changing IFilter, but the name base doesn't make sense here. 

Maybe IFilterUnion or something more semantic? Usually if something is hard to name there is design problem.

You notice above the union type name for `IFilterTarget` took the generic form of the three, which is what `IFilter` would have done for these types.  Since that's already used we are kind of stuck.